### ISSUE TITLE
[Feature request] Extend the hidden channels to the search

### ISSUE THREAD

**Issue by ArtemPindrus** [futo-org/grayjay-android#410]
There is a possibility to hide channels/videos from home. A switch to extend hiding to the search would be useful.

**Comment by Kelvin-FUTO** [comment]
While this is easily done, It would likely need to be an optional setting, because for most users this would be unexpected behavior.

**Comment by priyanshujoshi19** [comment]
yeah i wanted had posted this on [zulip](https://chat.futo.org/#narrow/stream/46-Grayjay/topic/Add.20Ability.20to.20block.20creators.20and.20videos), even if videos are hidden from its not enough, there aren't even videos on home anyway, the search is the distracting part, seeing uncomfortable videos/creators just take the attention, this feature should have been on youtube long back.

as for optional setting, is it not possible that to have an icon in the search menu where you could switch to see the blocked content based on you current search (if any), or maybe just a toggle.

**Comment by PacoBell** [comment]
Please prioritize channel hiding in search. The AI slop problem is accelerating and only getting worse. It's becoming very difficult to search intended content on YouTube.

**Comment by Kelvin-FUTO** [comment]
I've implemented a toggle that allows you to hide the same results in search (both creators and videos) for the next android release. 
It will be a setting in next release you have to toggle on "Hide hidden from home in search"

**Comment by Zvonimir-FUTO** [comment]
Added in v284.
